.github/workflows/publish-registry.yml

```diff
@@ -50,9 +48,12 @@ jobs:
           mkdir -p dist
           cp src/registry.json dist/registry.json
           cp src/signature.json dist/signature.json
-      - name: Deploy registry
-        uses: peaceiris/actions-gh-pages@de7ea6f8efb354206b205ef54722213d99067935
+      - name: configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
```

Comment:
This will need to be pinned to a certain commit. We use a whitelist within the MetaMask organisation, and only whitelist commits, not version ranges.

Reply:
Pin to commit SHA, thanks!